cmd/heal.go

```diff
@@ -0,0 +1,359 @@
+package cmd
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/minio/minio-go/v6"
+	"github.com/minio/minio-go/v6/pkg/encrypt"
+	"github.com/minio/minio/pkg/sync/errgroup"
+	"github.com/minio/radio/cmd/logger"
+	"github.com/vmihailenco/msgpack"
+)
+
+type operation string
+
+var (
+	opPutObject         operation = "PutObject"
+	opDeleteObject      operation = "DeleteObject"
+	opCopyObject        operation = "CopyObject"
+	opCompleteMultipart operation = "CompleteMultipart"
+)
+
+// ErrorLog struct for errors
+type ErrorLog struct {
+	Timestamp            time.Time          `json:"timestamp"`
+	Bucket               string             `json:"bucket"`
+	Object               string             `json:"object"`
+	ErrClientID          string             `json:"err_client_id"`
+	SrcClientID          string             `json:"src_client_id"`
+	Op                   operation          `json:"operation"`
+	ETag                 string             `json:"etag"`
+	DstBucket            string             `json:"dst_bucket"`
+	DstObject            string             `json:"dst_object"`
+	RadioTagID           string             `json:"radio_tag_id"`
+	UserMeta             map[string]string  `json:"user_meta"`
+	ServerSideEncryption encrypt.ServerSide `json:"sse"`
+}
+
+const timeFormat = "2006-01-02T15:04:05.000Z"
```

Comment:
the filename can be unix epoch time here @poornas we don't need this type of format any particular purpose this was used?